app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -174,7 +174,7 @@ def check_policy_prevent(policy_event, event_target, cb_method)
       :class_name  => self.class.to_s,
       :instance_id => id,
       :method_name => :check_policy_prevent_callback,
-      :args        => [cb_method, event_target],
+      :args        => [cb_method, event_target.class.name, event_target.id],
```

Comment:
Maybe it's too complicated to do, but... since travis collects logs (and pastes them in the end)
would be possible to grep them in a test that runs in the end?

This specific problem would have been caught by something like that (I think, I know that flow is tested)